openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/weighting/HeatStressWeighting.java

```diff
@@ -29,9 +29,11 @@ public HeatStressWeighting(FlagEncoder encoder, PMap map, GraphStorage graphStor
 
     @Override
     public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevOrNextEdgeId) {
+        // TODO: this is a short term solution to be adapted when
+        //       AdditionWeighting will be replaced
         if (heatStressStorage != null) {
             int stressLevel = heatStressStorage.getEdgeValue(EdgeIteratorStateHelper.getOriginalEdge(edgeState), columnIndex, buffer);
-            return stressLevel * weightingFactor / 100;
+            return stressLevel * weightingFactor / 100 + 1.0;
```

Comment:
Is this to get a value range from [-1,1]? Wasn't the value range of the shadow data in the csv [1,2]?

Reply:
The range in the data file should be in the interval [-1,1] like we wish to have it in the long term. The +1.0 moves it into the interval [0,2] to compensate for the current implementation of `AdditionWeighting`.